src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
```

Comment:
Missing font-weight property for the Avenir Heavy font. Add `font-weight: 600;` or any appropriate value.